examples/training/classification/imagenet/basic_training.py

```diff
@@ -148,6 +153,7 @@ def augment(img, label):
         include_top=True,
         classes=CLASSES,
         input_shape=IMAGE_SIZE + (3,),
+        **eval(FLAGS.model_kwargs),
```

Comment:
can we json.loads this instead?

Reply:
Since JSON requires double-quote tags, I think this is pretty unintuitive from the CLI because you'd have to use single-quote outer quotes with double-quote inner quotes, whereas this way it works with either approach

Reply:
ok, fair enough.  Great point